Rubberduck.Core/UI/Refactorings/ReorderParameters/ReorderParametersPresenter.cs

```diff
@@ -1,18 +1,14 @@
 ﻿using System.Collections.ObjectModel;
 using System.Linq;
-using System.Windows.Forms;
 using Rubberduck.Resources;
-using Rubberduck.UI;
+using Rubberduck.Interaction;
 using Rubberduck.UI.Refactorings;
 using Rubberduck.UI.Refactorings.ReorderParameters;
+using System.Windows.Forms;
```

Comment:
Same question RE: the reference

Reply:
Same reason as in RemoveParametersPresenter